CONTRIBUTING.md

```diff
@@ -154,6 +166,20 @@ Prior to making a new release, there needs to be testing that the
 current Helm chart works with the executable behavior that will
 appear in the new release.  
 
+## GitHub Action Reference Discipline
```

Comment:
Why is this PR inserting material that already exists in this file (https://github.com/kubestellar/kubestellar/blob/87ed2495649c87eab669fe0d413fb15f5e2da41e/CONTRIBUTING.md?plain=1#L112-L123)? Something is going wrong here.